packages/app-check-interop-types/index.d.ts

```diff
@@ -20,6 +20,10 @@ export interface FirebaseAppCheckInternal {
   // is present. Returns null if no token is present and no token requests are in-flight.
   getToken(forceRefresh?: boolean): Promise<AppCheckTokenResult>;
 
+  // Gets a limited use Firebase App Check token. This method should be used
```

Comment:
nit: Is this for the internal interop only? If so, I think this comment was meant for 3P developers -- here we can just say something like
```
Always returns a fresh limited-use token suitable for Replay Protection.
The returned token must be used and consumed as soon as possible.
```